slp_abacus/slp_abacus/load/abacus_loader.py

```diff
@@ -0,0 +1,69 @@
+import json
+import logging
+import yaml
+
+from sl_util.sl_util.file_utils import read_byte_data
+from slp_abacus.slp_abacus.objects.diagram_objects import Diagram, DiagramComponent, DiagramRepresentation
+from slp_base import LoadingDiagramFileError
+from slp_base.slp_base.provider_loader import ProviderLoader
+
+logger = logging.getLogger(__name__)
+
+
+class AbacusLoader(ProviderLoader):
+    """
+    Builder for an Abacus class from the json data
+    """
+
+    def __init__(self, project_id: str, abacus_source, mapping_files: [bytes]):
+        self.project_id = project_id
+        self.abacus_source = abacus_source
+        self.mapping_files = mapping_files
+
+        self._diagram = None
+
+    def load(self):
+        try:
+            # Parse JSON and YAML data
+            out_connections = json.loads(read_byte_data(self.abacus_source))["OutConnections"]
+
+            abacus_mapping: str = read_byte_data(self.mapping_files[0])
```

Comment:
In this class, we are still not using this mappings, so we don't need to load this source.